coderdojochi/forms.py

```diff
@@ -3,10 +3,17 @@
 from django.forms import Form, ModelForm, FileField, ValidationError
 from django.utils.html import format_html
 from django.utils.safestring import mark_safe
-from coderdojochi.models import Mentor, Guardian, Student
+from coderdojochi.models import Mentor, Guardian, Student, RaceEthnicity
 
 import html5.forms.widgets as html5_widgets
 
+school_type_choices = (
+("Public", "Public"),
+("Charter", "Charter"),
+("Private", "Private"),
+("Homeschool", "Homeschool")
+)
+
```

Comment:
Super small comment but I am going to be implementing  PEP8 standards throughout the codebase here shortly so you'd want to indent the options in this tuple.  If your using Sublime, [this](https://github.com/SublimeLinter/SublimeLinter-pep8) is a great tool.